### PR TITLE
reorder everything that it makes more sense

### PR DESCRIPTION
**Pull request by mowolf**
Please check the image download thing. There is still too much space but I don't know where it comes from.

- added proper y margins
- reordered everything

### CODE REVIEW

File: components/ChatVisualization/ChatVisualization.vue [hunk 1]
```diff
@@ -1,10 +1,93 @@
 <template>
   <v-col class="my-4 mb-16">
+    <div class="text-h1 font-weight-bold">Your Full Chat</div>
+    <v-divider class="py-5" />
+    <!-- this could be refactored into a component -->
+    <v-row justify="center" id="payButton">
```
Comment (fellnerse) [inline components/ChatVisualization/ChatVisualization.vue:6]: yes pls

**Review by fellnerse — CHANGES_REQUESTED** [review]
looks nice! i think i would prefere the funfacts above the graphs;
in groupchats we could hide after lets say 4 and if you want to see mehr btn to load more sth like this?

File: components/ChatVisualization/ChatVisualization.vue (hunk 1 above)
Comment (PKehnel) [reply components/ChatVisualization/ChatVisualization.vue:6]: would also prefer the funfacts to move up

**Review by mowolf — COMMENTED** [review]
thanks :)

**Review by Adrian-Thiesen — APPROVED** [review]

**Review by Adrian-Thiesen — APPROVED** [review]

**Review by Adrian-Thiesen — APPROVED** [review]

**Pull request merged**